config/routes.rb

```diff
@@ -36,7 +36,7 @@
 
   devise_for :users
 
-  resources :users, only: [:index, :show, :edit, :update]
+  resources :users, only: [:index, :show, :edit, :update], constraints: { id: /[^\/]+/ }
```

Comment:
@brugidou In the future we could extend the github-connector to support other formats (JSON etc.). A [quick search](http://stackoverflow.com/questions/6719797/need-a-rails-route-with-a-possible-period-in-the-id-but-also-retain-the-option) shows that we can explicitly specify the `:format` option which would only be HTML today. If you wanted to go ahead and add that it definitely makes sense to support dots in usernames.